src/Kafka/Kafka.purs

```diff
@@ -0,0 +1,133 @@
+module Kafka.Kafka
+  ( LogLevel(..)
+  , Kafka
+  , KafkaConfig
+  , KafkaConfigBrokers(..)
+  , newKafka
+  ) where
+
+import Prelude
+
+import Control.Promise as Control.Promise
+import Data.Maybe as Data.Maybe
+import Effect as Effect
+import Effect.Aff as Effect.Aff
+import Effect.Uncurried as Effect.Uncurried
+import Kafka.FFI as Kafka.FFI
+import Untagged.Union as Untagged.Union
+
+-- | https://github.com/tulios/kafkajs/blob/dcee6971c4a739ebb02d9279f68155e3945c50f7/types/index.d.ts#L17
+-- |
+-- | * `BrokersFunction = () => string[] | Promise<string[]>`
+type BrokerFunctionImpl =
+  Effect.Effect (Array String)
+    Untagged.Union.|+| Effect.Effect (Control.Promise.Promise (Array String))
+
+-- | https://github.com/tulios/kafkajs/blob/dcee6971c4a739ebb02d9279f68155e3945c50f7/types/index.d.ts#L9
+foreign import data Kafka :: Type
+
+-- | * `brokers`
+-- | * `clientId`
+-- |   * A logical identifier of an application to be included in
+-- |     server-side request logging.
+-- |   * see [Client Id](https://kafka.js.org/docs/configuration#client-id)
+type KafkaConfig =
+  { brokers :: KafkaConfigBrokers
+  , clientId :: Data.Maybe.Maybe String
```

Comment:
It'd be better to add field-level documentation above the fields instead of above the type, as it makes it easier for developers to see the documentation and to remember to update it.

Reply:
I thought about that but does Pursuit doc support interleaving comments between Record fields?

Reply:
Hm so are we planning on uploading this to Pursuit?

Reply:
We could but not until we upgrade to PS 0.15.